airflow/Dockerfile

```diff
@@ -43,9 +43,8 @@ RUN microdnf update && \
         # Needed to modify the SBOM
         jq && \
     microdnf clean all && \
-    rm -rf /var/cache/yum
-
-RUN <<EOF
+    rm -rf /var/cache/yum && \
+bash <<EOF
```

Comment:
Why are we now using both heredocs and individual RUN statements?

Reply:
Sorry, I didn't fully get that. The only part that I see changed is now re-using the `bash` instead of spawning 2 separate bash invocations. The result (one RUN instead of two) should save disk space in the runners.

Reply:
I'm against that as mentioned last week as it just introduces another concept. Just do what the other operators do even if it's not as pretty otherwise we have more inconsistencies.

Reply:
> Sorry, I didn't fully get that. The only part that I see changed is now re-using the `bash` instead of spawning 2 separate bash invocations. The result (one RUN instead of two) should save disk space in the runners.

Yes, that one RUN command can also be simplified by putting the entire script in a single heredoc string like in this [example](https://github.com/stackabletech/docker-images/blob/812fdcf06c901654ba1facb95ef1a53d9deec0e6/hbase/Dockerfile#L150) 

It saves space *and*  is easier to read.

Reply:
in other words, why is the `microdnf` call outside the <<EOF string ?

Reply:
Reverted the optimization again in 296d1b905d8acea690bfedd0420b5fedb0525558, so that this Pr only focuses on adding some comments